Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -247,11 +259,13 @@ - (void)stopAnimating {
     RunSynchronouslyOnMainThread(^{
         if (_isAnimating) {
             _isAnimating = NO;
-        }
 
-        _progressRing.IsActive(false);
-        if (_hidesWhenStopped) {
-            [self setHidden:YES];
+            if (_hidesWhenStopped) {
+                // Make our view transparent, rather than hidden, to ensure space is reserved in the layout.
+                [self setAlpha:0.0];
```

Comment:
In iOS the background is set on the entire UIActivityIndicatorView, and the entire view hides when stopped (not just the indicator itself). So we actually don't want to show the background when the ProgressRing is hidden. Currently the background and frame size are both modifying the Grid that contains the ProgressRing, which is correct from an iOS standpoint. 

---
In reply to: [113292895](https://github.com/Microsoft/WinObjC/pull/2608#discussion_r113292895) [](ancestors = 113292895,113288024,113080688)